@W-18779406 - T/commerce/spark/storelocatorfunctionality

aditek-sf opened the pull request:
<!--- Provide a short summary of your changes in the Title field above -->

# Description

<!--- A longer summary of your changes, including: a description of the issue that you’re addressing, a list of required dependencies (if applicable), and any other relevant context. -->

# Types of Changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] **Bug fix** (non-breaking change that fixes an issue)
- [ ] **New feature** (non-breaking change that adds functionality)
- [ ] **Documentation update**
- [ ] **Breaking change** (could cause existing functionality to not work as expected)
- [ ] **Other changes** (non-breaking changes that does not fit any of the above)

> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

- (step1)

# Checklists

<!--- Enter an `x` in all the boxes that apply. -->
<!--- If you’re unsure about any of these, don’t hesitate to ask. We’re here to help! -->

## General

- [ ] Changes are covered by test cases
- [ ] CHANGELOG.md updated with a short description of changes (_not_ required for documentation updates)

## Accessibility Compliance

You must check off all items in **one** of the follow two lists:

- [ ] There are no changes to UI

_or..._

- [ ] Changes were tested with a Screen Reader (iOS VoiceOver or Android Talkback) and had no issues
- [ ] Changes comply with [WCAG 2.0 guidelines levels A and AA](https://www.wuhcag.com/wcag-checklist/)
- [ ] Changes to common UI patterns and interactions comply with [WAI-ARIA best practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## Localization

- [ ] Changes include a UI text update in the Retail React App (which requires translation)

sf-henry-semaganda committed:
shelving changes

sf-henry-semaganda committed:
restore store selection functionality

sf-henry-semaganda committed:
fix named import in invemntory filter

sf-henry-semaganda committed:
fix annoying typo sigh

sf-henry-semaganda committed:
fix linting

sf-henry-semaganda committed:
fix default country code constant

sf-henry-semaganda committed:
update test file as well

sf-henry-semaganda committed:
update package.json

sf-henry-semaganda committed:
Merge branch 'feature/shop-in-store' into W-18762060_newStoreLocator

Signed-off-by: sf-henry-semaganda <henry.semaganda@salesforce.com>

sf-henry-semaganda committed:
upadte linting

sf-henry-semaganda committed:
update

sf-henry-semaganda committed:
updated import to new store locator

sf-henry-semaganda committed:
update linting

sf-henry-semaganda committed:
fix failing tests

sf-henry-semaganda committed:
update

sf-henry-semaganda committed:
update 2

sf-henry-semaganda committed:
fix failing test

sf-henry-semaganda committed:
Merge branch 'feature/shop-in-store' into W-18762060_newStoreLocator

Signed-off-by: sf-henry-semaganda <henry.semaganda@salesforce.com>

sf-henry-semaganda committed:
Update en-XA.json

Signed-off-by: sf-henry-semaganda <henry.semaganda@salesforce.com>

sf-henry-semaganda committed:
Update en-XA.json

Signed-off-by: sf-henry-semaganda <henry.semaganda@salesforce.com>

sf-henry-semaganda committed:
Update en-XA.json

Signed-off-by: sf-henry-semaganda <henry.semaganda@salesforce.com>

sf-henry-semaganda committed:
Update en-XA.json

Signed-off-by: sf-henry-semaganda <henry.semaganda@salesforce.com>

sf-henry-semaganda committed:
Update en-XA.json

Signed-off-by: sf-henry-semaganda <henry.semaganda@salesforce.com>

sf-henry-semaganda committed:
Update en-XA.json

Signed-off-by: sf-henry-semaganda <henry.semaganda@salesforce.com>

sf-henry-semaganda committed:
Update en-XA.json

Signed-off-by: sf-henry-semaganda <henry.semaganda@salesforce.com>

sf-henry-semaganda committed:
fix file

aditek-sf committed:
skip changelog

cc-prodsec commented:
![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/pr-banner-default.svg)

### :tada: **Snyk checks have passed. No issues have been found so far.**

:white_check_mark: **security/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/cc-mobify-sdk/pr-checks/5ea9dcd7-584b-4352-b7f9-b97d0d800067?refs=58c5d965a3bca74d00462008edcda439a95a2780&source=prChecksComment)

:white_check_mark: **license/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/cc-mobify-sdk/pr-checks/5ea9dcd7-584b-4352-b7f9-b97d0d800067/license?refs=58c5d965a3bca74d00462008edcda439a95a2780&source=prChecksComment)

aditek-sf committed:
skip changelog

aditek-sf committed:
Update pseudo translations for store locator functionality

aditek-sf committed:
Remove conflicting store_locator translations to match develop branch

- Remove store_locator.action.* translations
- Remove store_locator.description.loading_locations, no_locations, or, viewing_*
- Remove store_locator.dropdown.* translations
- Remove store_locator.error.* translations
- Remove store_locator.field.* and store_locator.pagination.* translations
- Remove store_locator.title translation
- Keep store_locator.description.away, email, phone for compatibility
- Recompile all translation files to match expected CI state

aditek-sf committed:
Added correct imports

aditek-sf committed:
Fixed export/import issues
